apis/management.cattle.io/v3/cluster_scan_types.go

```diff
@@ -11,9 +11,29 @@ import (
 const (
 	ClusterScanConditionCreated   condition.Cond = "Created"
 	ClusterScanConditionCompleted condition.Cond = "Completed"
+
+	ClusterScanTypeCIS         = "cis"
+	DefaultNamespaceForCis     = "security-scan"
+	DefaultSonobuoyPodName     = "security-scan-runner"
+	ConfigMapNameForUserConfig = "security-scan-cfg"
+
+	RunCISScanAnnotation         = "field.cattle.io/runCisScan"
+	SonobuoyCompletionAnnotation = "field.cattle.io/sonobuoyDone"
+	CisHelmChartOwner            = "field.cattle.io/clusterScanOwner"
 )
 
+type CisScanConfig struct {
```

Comment:
should it be CIS?

Reply:
Or the other way around. If generator is unhappy with the caps, we should switch to camel case in other places.